StatBot/Commands/Stats.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.Bot.Schema;
+using StatBot.Interfaces;
+using StatBot.Models;
+using System;
+using System.Collections.Generic;
+
+namespace StatBot.Commands
+{
+    [Serializable]
+    public class Stats : ITool
+    {
+        public string Description { get; set; }
+        public List<string> CommandsName { get; set; }
+        public bool IsAdmin { get; set; }
+
+        public Activity Run(Activity activity)
+        {
+            string StatNames = "";
+            if (activity?.Conversation != null)
+            {
+                foreach (var stat in DataModel.Statistics)
+                {
+                    if (stat.IsActive)
+                        StatNames += $"{stat.Id}. {stat.Name}\n";
+                }
+            }
+            activity.Text = StatNames;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') 'activity' is null on at least one execution path. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2259)